tests/js/spec/views/settings/account/notifications/notificationSettingsByOrganizations.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import {mountWithTheme} from 'sentry-test/enzyme';
+import {initializeOrg} from 'sentry-test/initializeOrg';
+
+import {NotificationSettingsObject} from 'app/views/settings/account/notifications/constants';
+import NotificationSettingsByOrganization from 'app/views/settings/account/notifications/notificationSettingsByOrganization';
+
+const createWrapper = (notificationSettings: NotificationSettingsObject) => {
+  const {organization, routerContext} = initializeOrg();
+  return mountWithTheme(
+    <NotificationSettingsByOrganization
+      notificationType="alerts"
+      notificationSettings={notificationSettings}
+      organizations={[organization]}
+      onChange={jest.fn()}
+    />,
+    routerContext
+  );
+};
+
+describe('NotificationSettingsByOrganization', function () {
```

Comment:
can you add a test that the orgs don't render when the setting is turned off?

Reply:
I updated the `NotificationSettingsByType` test to make that no other parents render. We can't test it in `NotificationSettingsByOrganization` since it will never be rendered if there are no settings.